media/css/firefox/new/protocol/mobile-sync.scss

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../../../protocol/css/includes/lib';
+
+/* -------------------------------------------------------------------------- */
+// Download thank you page for quantum lang file and trailhead design
```

Comment:
Might this page look nice with the content centered? (up to you, just a suggestion)

![image](https://user-images.githubusercontent.com/400117/75529586-6ee71700-5a0b-11ea-91e1-96a37852b698.png)

Reply:
✅